mne/tests/test_bem.py

```diff
@@ -368,7 +368,6 @@ def test_fit_sphere_to_headshape():
     pytest.raises(TypeError, fit_sphere_to_headshape, 1, units='m')
 
 
-@requires_nibabel()
```

Comment:
I think this does still require `nibabel` to pass. If you don't have it, a warning will be triggered, and that warning will cause the test to fail. So I'd restore this